### PR TITLE
Fix community resource creation and display

### LINKED ISSUE #1732: Can not create community resource
Since 1.4.0 it's not possible to create a community resource anymore: the form/link goes to a standard resource creation.

### PR DESCRIPTION
**Pull request by abulte**
Fix #1732

### CODE REVIEW

File: js/front/dataset/index.js [hunk 1]
```diff
@@ -122,7 +122,11 @@ new Vue({
             location.hash = `resource${communityPrefix}-${id}`;
             const modal = this.$modal(
                 ResourceModal,
-                {datasetId: this.dataset['@id'], resource: this.resourceFromJsonLd(resourceJsonLd)}
+                {
+                    datasetId: this.dataset['@id'],
+                    resource: this.resourceFromJsonLd(resourceJsonLd),
+                    isCommunity: isCommunity,
```
Comment (noirbizarre) [inline js/front/dataset/index.js]: Isn't it possible to read this boolean from the `resource` paramter attributes ?

Comment (abulte) [reply js/front/dataset/index.js]: Probably yes 👍

Comment (abulte) [reply js/front/dataset/index.js]: Nope, nothing in there.

Comment (noirbizarre) [reply js/front/dataset/index.js]: OK, so LGTM
(maybe we should think about exposing this attribute in resources)

**Pull request merged**